lib/signs_ui_web/router.ex

```diff
@@ -57,13 +57,24 @@ defmodule SignsUiWeb.Router do
       :browser,
       :auth,
       :ensure_auth,
-      :ensure_signs_ui_group,
       :put_user_token,
       :put_read_only_view
     ])
 
     get("/viewer", MessagesController, :index)
     get("/scu", ScuController, :index)
+  end
+
+  scope "/", SignsUiWeb do
+    pipe_through([
+      :redirect_prod_http,
+      :browser,
+      :auth,
+      :ensure_auth,
+      :ensure_signs_ui_admin_group,
```

Comment:
The `ScuController` is currently doing its own auth check, but this seems like a better approach. Now that this is in place, I think we can remove the auth check there.